resources/resolver.go

```diff
@@ -1025,15 +1026,6 @@ func (rsrcs *Resources) ResolveHF(hfConfig *HFConfig) (err error) {
 		return errors.New("could not resolve HFConfig")
 	}
 
-	// Llama 3 and other larger models will enclose eos and bos by default
-	if *hfConfig.VocabSize > math.MaxUint16+1 {
```

Comment:
We need to reconsider this. The behavioral detection is important, as part of the requirement of `gpt_bpe` is that it be able to use LLama models straight from Huggingface.